### PR TITLE
graph docs update for TLE deprecation

### PR DESCRIPTION
**Pull request by awang119**

### CODE REVIEW

File: api-reference/beta/api/application-post-onlinemeetings.md [hunk 1]
```diff
@@ -190,7 +186,7 @@ Content-Type: application/json
     }
 ```
 
-### Example 2: Create a Microsoft Teams live event with a user token
+### Example 2: Create a Microsoft Teams live event with a user token (deprecated)
```
Comment (frankpeng7) [inline api-reference/beta/api/application-post-onlinemeetings.md:189]: I would actually suggest just removing this example if there's no guideline saying not to

File: api-reference/beta/api/onlinemeeting-get.md [hunk 2]
```diff
@@ -18,13 +18,17 @@ Retrieve the properties and relationships of an [onlineMeeting](../resources/onl
...
 
 - Get details of an onlineMeeting using [videoTeleconferenceId](#example-1-retrieve-an-online-meeting-by-videoteleconferenceid), [meeting ID](#example-2-retrieve-an-online-meeting-by-meeting-id), [joinWebURL](#example-3-retrieve-an-online-meeting-by-joinweburl), or [joinMeetingId](#example-4-retrieve-an-online-meeting-by-joinmeetingid).
-- Use the `/attendeeReport` path to get the attendee report of a [Microsoft Teams live event](/microsoftteams/teams-live-events/what-are-teams-live-events) in the form of a download link, as shown in [example 5](#example-5-fetch-the-attendee-report-of-a-teams-live-event).
+- Use the `/attendeeReport` path to get the attendee report of a [Microsoft Teams live event](/microsoftteams/teams-live-events/what-are-teams-live-events) in the form of a download link, as shown in [example 5](#example-5-fetch-the-attendee-report-of-a-teams-live-event). **(deprecated)**
 - Use the `/recording` and `/alternativeRecording` paths to get the recordings of a [Teams live event](/microsoftteams/teams-live-events/what-are-teams-live-events) in the form of a download link, as shown in [example 6](#example-6-fetch-the-recording-of-a-teams-live-event).
```
Comment (frankpeng7) [inline api-reference/beta/api/onlinemeeting-get.md:22]: the /recording and /alternativeRecording paths are also for TLE only

File: api-reference/beta/resources/onlinemeeting.md [hunk 3]
```diff
@@ -142,14 +146,12 @@ This resource supports subscribing to [change notifications](/graph/webhooks). F
   "allowedPresenters": "String",
   "alternativeRecording": "Stream",
```
Comment (frankpeng7) [inline api-reference/beta/resources/onlinemeeting.md:147]: this needs to be removed too

Comment (frankpeng7) [reply api-reference/beta/resources/onlinemeeting.md:147]: as well as "recording" in L#160/162

File: api-reference/beta/api/application-post-onlinemeetings.md (hunk 1 above)
Comment (awang119) [reply api-reference/beta/api/application-post-onlinemeetings.md:189]: @Lauragra what are your thoughts about this?

File: api-reference/beta/resources/onlinemeeting.md (hunk 3 above)
Comment (awang119) [reply api-reference/beta/resources/onlinemeeting.md]: @frankpeng7  you mean just "alternativeRecording" right? "allowedPresenters" shouldn't be deprecated

File: api-reference/beta/api/application-post-onlinemeetings.md (hunk 1 above)
Comment (Lauragra) [reply api-reference/beta/api/application-post-onlinemeetings.md:189]: @awang119, it makes sense to me, thanks.

**Pull request merged**